lib/plugins/features/movies/index.js

```diff
@@ -20,6 +20,16 @@ exports.register = (server, options, next) => {
   }]);
 
   server.route([{
+    method: 'POST',
+    path: '/add_location',
```

Comment:
According to our [API style guide](https://github.com/lob/wiki/blob/master/style-guides/api.md), and more specifically [this article](http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#restful), we try to keep the URLs as RESTful as possible. Since we are dealing with the relationship between movies and locations, we can do `/movies/{id}/locations`.